core/src/test/java/io/forestframework/core/http/routing/MediaTypeNegotiatingIntegrationTest.java

```diff
@@ -0,0 +1,8 @@
+package io.forestframework.core.http.routing;
```

Comment:
Unused class.

Reply:
Do you think it's better to remove `MediaTypeNegotiatingIntegrationTest.java`? it's never used.

Reply:
yes, please.

Reply:
will do.